### ISSUE TITLE
BlockPlaceEvent error on Angel Chest v.7.8.0

### ISSUE THREAD

**Issue by ilfarpro** [JEFF-Media-GbR/issue-tracker#3]
```
- [Sat 11:24:49 ERROR Minecraft] Could not pass event BlockPlaceEvent to AngelChest v7.8.0
java.lang.NullPointerException
    at java.base/java.util.Objects.requireNonNull(Objects.java:208)
    at AngelChest-7.8.0.jar//de.jeff_media.angelchest.jefflib.PDCUtils.has(PDCUtils.java:329)
    at AngelChest-7.8.0.jar//de.jeff_media.angelchest.Com2.onPlaceAngelChestItem(sa:954)
    at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor128.execute(Unknown Source)
    at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75)
    at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76)
    at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
    at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:665)
    at CMI_9.1.4.1.jar//com.Zrips.CMI.Modules.AStand.ArmorStandManager.canBuild(ArmorStandManager.java:1432)
    at CMI_9.1.4.1.jar//com.Zrips.CMI.Modules.AStand.ArmorStandManager.changeAngle(ArmorStandManager.java:1522)
    at CMI_9.1.4.1.jar//com.Zrips.CMI.Modules.AStand.ArmorStandManager.changeAngle(ArmorStandManager.java:1468)
    at CMI_9.1.4.1.jar//com.Zrips.CMI.Modules.AStand.ArmorStandManager$30.click(ArmorStandManager.java:1127)
    at CMILib1.1.2.5.jar//net.Zrips.CMILib.GUI.GUIManager.processClick(GUIManager.java:263)
    at CMILib1.1.2.5.jar//net.Zrips.CMILib.GUI.GUIListener.onInventoryClick(GUIListener.java:220)
    at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor259.execute(Unknown Source)
    at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75)
    at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76)
    at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
    at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:665)
    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleContainerClick(ServerGamePacketListenerImpl.java:2946)
    at net.minecraft.network.protocol.game.Ser
```

**Comment by mfnalex** [comment]
Hi, is this the free or paid version of AngelChest?

**Comment by ilfarpro** [comment]
Hi! It's free version of AngelChest

**Comment by mfnalex** [comment]
It seems like CMI is calling an event that returns "null" in a method annotated with NotNull. I'll add a workaround for this in AngelChest 7.9.0 which will be released today or tomorrow

**Comment by ilfarpro** [comment]
OK, thank you!

**Comment by mfnalex** [comment]
This should be fixed in 7.9.0, which will be upload in a few minutes. If the error still occurs, pls reopen this issue and send the new stacktrace again
